### PR TITLE
Added new actions and fixed data ingestion playbook for Qradar

### PR DESCRIPTION
**Pull request by cs-omkar-kumbhar**
Description:

Fixed bug in data ingestion playbook of Qradar

Mantis Link: https://mantis.fortinet.com/bug_view_page.php?bug_id=0871550

- New actions:
    - Get assets properties: Get a list of available asset property types that can be used with assets update operation
    - Get assets: Get a list of available assets
    - Update asset: Update one or multiple attributes of a specific asset defined by Asset ID
    - Get cases: Get a list of Qradar cases
    - Create case: Create a new case on Qradar
    - Get reference tables: Get a list of Reference Tables
    - Get table elements: Get records details of a reference table defined by name
    - Add table element: Add or update an element in a reference table
    - Delete table element: Delete element from a reference table
    - Delete reference table: Delete a Reference Table or Purge its content

Verified all 28 actions.

### CODE REVIEW

File: qradar/info.json [hunk 1]
```diff
@@ -677,6 +677,565 @@
           }
         }
       ]
+    },
+    {
+      "operation": "get_assets_properties",
+      "title": "Get Assets Properties",
+      "description": "Get a list of available asset property types that can be used with assets update operation.",
+      "category": "enrichment",
```
Comment (cs-prerna-joshi) [inline qradar/info.json]:
```suggestion
      "category": "investigation",
```

File: qradar/info.json [hunk 2]
```diff
@@ -677,6 +677,565 @@
...
+      "annotation": "create_case",
+      "output_schema": {
+   "case_id":"",
+   "name":"",
+   "id":"",
+   "state":"",
+   "assigned_to":[
+
+   ]
```
Comment (cs-prerna-joshi) [inline qradar/info.json]: reformat the file.

File: qradar/info.json [hunk 3]
```diff
@@ -677,6 +677,565 @@
...
+        {
+          "title": "Reference Table Name",
+          "description": "The name of the reference table to delete or purge",
+          "tooltip": "The name of the reference table to delete or purge",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "path.name",
+          "placeholder": "",
```
Comment (cs-prerna-joshi) [inline qradar/info.json]: remove if not needed.

File: qradar/release_notes.md [hunk 4]
```diff
@@ -1,4 +1,13 @@
-#### What's Fixed
-  
-- Updated jinja .timestamp to .int_timestamp in the data ingestion playbooks.
-- Added the "PATCH" option to the "Request Method" parameter of the "Invoke QRadar REST API" operation.
\ No newline at end of file
+#### What's new in 1.6.0
```
Comment (cs-prerna-joshi) [inline qradar/release_notes.md]:
```suggestion
#### What's Improved
```

File: qradar/release_notes.md [hunk 5]
```diff
@@ -1,4 +1,13 @@
...
+
+- New actions:
```
Comment (cs-prerna-joshi) [inline qradar/release_notes.md]:
```suggestion
- Added the following new operations and playbooks:
```

File: qradar/release_notes.md [hunk 6]
```diff
@@ -1,4 +1,13 @@
...
+    - Get assets properties: Get a list of available asset property types that can be used with assets update operation
```
Comment (cs-prerna-joshi) [inline qradar/release_notes.md]: just keep the Action name. Description is not needed.

File: qradar/info.json (hunk 1 above)
Comment (cs-omkar-kumbhar) [reply qradar/info.json]: done

File: qradar/info.json (hunk 2 above)
Comment (cs-omkar-kumbhar) [reply qradar/info.json]: done

File: qradar/info.json (hunk 3 above)
Comment (cs-omkar-kumbhar) [reply qradar/info.json]: done

File: qradar/release_notes.md (hunk 4 above)
Comment (cs-omkar-kumbhar) [reply qradar/release_notes.md]: done

File: qradar/release_notes.md (hunk 5 above)
Comment (cs-omkar-kumbhar) [reply qradar/release_notes.md]: done

File: qradar/release_notes.md (hunk 6 above)
Comment (cs-omkar-kumbhar) [reply qradar/release_notes.md]: done

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**